package.json

```diff
@@ -15,7 +15,7 @@
     "dynamics.js": "^1.1.5",
     "gsap": "^3.9.0",
     "vitepress": "1.0.0-rc.31",
-    "vue": "^3.4.0-beta.4"
+    "vue": "^3.4.0-rc.2"
```

Comment:
`vitepress` and `vue` should be bumped to `rc3`. https://github.com/vuejs/docs/commit/810409f0138250122f865752b34d0e8610377018#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R17-R18